docs/install.md

```diff
@@ -132,14 +131,6 @@ In a nutshell:
     ./open_spiel/scripts/build_and_run_tests.sh
     ```
 
-    **Building and testing using PIP**
-
-    ```bash
-    python3 -m pip install .
```

Comment:
Leave this section because OpenSpiel can still be installed using pip this way. Just remove the next two lines